CSharp/BatchExplorer/Converters/InvertedBooleanConverter.cs

```diff
@@ -12,18 +12,28 @@ namespace Microsoft.Azure.BatchExplorer.Converters
     public class InvertedBooleanConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
-        {
-            if (targetType != typeof (bool))
-            {
-                throw new InvalidOperationException("The target must be a bool.");
-            }
-
-            return !(bool)value;
-        }
+        {
+            return _Convert(value, targetType);
+        }       
 
         public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
Convert and ConvertBack method come from the interface IValueConverter. So it's mandatory to define them.